internal/plugins/golang/v2/init.go

```diff
@@ -67,5 +62,11 @@ func (p *initPlugin) Run() error {
 
 // SDK plugin-specific scaffolds.
 func (p *initPlugin) run() error {
-	return utilplugins.UpdateMakefile(p.config)
+	if err := manifests.RunInit(p.config); err != nil {
+		return err
+	}
+	if err := scorecard.RunInit(p.config); err != nil {
+		return err
+	}
+	return nil
```

Comment:
I liked the manifest.RunInit idea I just think that it needs to be in the PostScaffold actions since is done after that in order to respect the idea of the cmdutil interface

Reply:
See https://github.com/operator-framework/operator-sdk/pull/3490#discussion_r459643492.